src/Microsoft.AspNetCore.Rewrite/Internal/CodeRules/RedirectRule.cs

```diff
@@ -54,7 +56,7 @@ public override void ApplyRule(RewriteContext context)
 
                 if (string.IsNullOrEmpty(newPath))
                 {
-                    response.Headers[HeaderNames.Location] = "/";
+                    response.Headers[HeaderNames.Location] = pathBase + '/';
```

Comment:
Pretty sure PathString can be null but not empty http://aspnetsource.azurewebsites.net/#Microsoft.AspNetCore.Http.Abstractions/PathString.cs,29